integration_test/extra_code_paths/path1/dummy_helper.erl

```diff
@@ -11,65 +16,73 @@ test_amoc_dist() ->
         Master = amoc_cluster:master_node(),
         Slaves = amoc_cluster:slave_nodes(),
         %% check the status of the nodes
-        disabled = rpc:call(Master, amoc_controller, get_status, []),
-        [{running, #{scenario := dummy_scenario}} = rpc:call(Node, amoc_controller, get_status, [])
-         || Node <- Slaves],
-        %% check user ids
-        {N1, Nodes1, Ids1, Max1} = get_users_info(Slaves),
-        true = N1 > 0,
-        N1 = Max1,
-        Ids1 = lists:seq(1, N1),
+        ?assertEqual(disabled, get_status(Master)),
+        [ ?assertMatch({running, #{scenario := dummy_scenario}}, get_status(Node)) || Node <- Slaves],
+        %% check user ids, users have all been started at the first two nodes
+        {N1, Max1, Nodes1, Ids1, Users1} = get_users_info(Slaves),
+        ?assert(N1 > 0),
+        ?assertEqual(N1, Max1, ?comment(Users1)),
+        ?assertEqual(Ids1, lists:seq(1, N1), ?comment(Users1)),
         [AddedNode] = Slaves -- Nodes1,
```

Comment:
why not assertEqual?

Reply:
Because then the value `AddedNode` is not captured and it is used a little bit below. Could do so nevertheless, only it would require two lines of code instead of one :)

Reply:
right